src/angular-version-comparer.ts

```diff
@@ -0,0 +1,28 @@
+import {AngularVersion} from './types/angular-version';
+
+/**
+ *
+ * Compares two angular versions, A and B. Based on their minor and major versions.
+ *
+ *
+ * @export
+ * @param {AngularVersion} versionA
+ * @param {AngularVersion} versionB
+ * @return {number}  > 0 if A is newer, 0 if they are the same version, < 0 if A is older
+ */
+export function angularVersionComparer(
+  versionA: AngularVersion,
+  versionB: AngularVersion
+): number {
+  // Gets the major and minor versions
+  const [majorA, minorA] = versionA.split('.');
+  const [majorB, minorB] = versionB.split('.');
+
+  // Obtains the major version difference between versions A and B
+  const majorDifference = Number.parseInt(majorA) - Number.parseInt(majorB);
+  // Obtains the minor version difference between versions A and B
+  const minorDifference = Number.parseInt(minorA) - Number.parseInt(minorB);
```

Comment:
suggestion: Pass the radix to be on the safe side.
```suggestion
  // Obtains the major version difference between versions A and B
  const majorDifference = Number.parseInt(majorA, 10) - Number.parseInt(majorB, 10);
  // Obtains the minor version difference between versions A and B
  const minorDifference = Number.parseInt(minorA, 10) - Number.parseInt(minorB, 10);
```

Reply:
@NachoVazquez
What do you think?

Reply:
Didn't I added it. It looks great BTW.